src/AutoMapper/ValueTransformerConfiguration.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Linq.Expressions;
+
+namespace AutoMapper
+{
+    public class ValueTransformerConfiguration
```

Comment:
It has function calls for IsMatch.  Would have to change it to a `Func<PropertyMap, bool>` for it to be a struct I think.